lib/ui/splash/splash_screen.dart

```diff
@@ -93,3 +120,109 @@ class _SplashScreenState extends ConsumerState<SplashScreen>
     super.dispose();
   }
 }
+
+class LoginForm extends ConsumerStatefulWidget {
```

Comment:
As we do this approach, I think we should rename this screen to `LoginScreen`, what do you think?

Reply:
@markgravity done in https://github.com/nimblehq/ic-flutter-avishek/pull/33/commits/12a37f81ec057868068a63706c8b15eda134e744.